src/components/Atoms/EditButton/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import styled from "styled-components";
+import COLOR from "../../../variables/color";
+import pencil from "../../../assets/svg/pencil.svg";
+
+const EditButton = ({ onClick }) => {
+  return (
+    <StyledEditButton onClick={onClick}>
+      <StyledCircle />
+      <StyledImg src={pencil} />
+    </StyledEditButton>
+  );
+};
+const StyledEditButton = styled.button`
+  width: 15px;
+  height: 15px;
+  background: rgba(255, 255, 255, 0);
```

Comment:
```suggestion
const StyledEditButton = styled.div`
  width: 15px;
  height: 15px;
```
background をこうしなければいけないのは、buttonのデフォルトスタイルがついているからだと思います。
html の button は form要素を発火させるための要素である一方、react開発においては html のformの機能をそのまま使って開発することはほとんどありません。
そのため、このようなコンポーネントは見た目だけボタンであればいいので、divで済ませるので問題ないと思います。